expression/builtin_json.go

```diff
@@ -136,6 +154,46 @@ func jsonModify(args []types.Datum, mt json.ModifyType, sc *variable.StatementCo
 	return d, nil
 }
 
+// jsonModify1 is similar with `jsonModify`, but for new framework.
+func jsonModify1(args []Expression, row []types.Datum, mt json.ModifyType, sc *variable.StatementContext) (res json.JSON, isNull bool, err error) {
```

Comment:
can we remove `jsonModify` ?

Reply:
After we remove `distsql/xeval` package.